addon/-private/system/model/record-data.js

```diff
@@ -25,12 +27,13 @@ export default class RecordData {
   }
 
   // PUBLIC API
-
+  // TODO IDENTIFIER RFC - RecordData should not be responsible for it's own identifier
   getResourceIdentifier() {
     return {
       id: this.id,
       type: this.modelName,
       clientId: this.clientId,
+      lid: this.clientId,
```

Comment:
I don't think we should merge this until an RFC discussion

Reply:
note that `lid` and `clientId` are both present here, sans deprecation. Users that have already implemented custom `RecordData`s returning only `clientId` will work fully as expected, but our default will now provide both. I'll be providing an RFC for identifiers for you to review over Thanksgiving break.